traffic_stats/influxdb_tools/sync/sync_test.go

```diff
@@ -0,0 +1,230 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	influx "github.com/influxdata/influxdb/client/v2"
+	"github.com/influxdata/influxdb/models"
+	. "github.com/smartystreets/goconvey/convey"
```

Comment:
I'd say clarity of tests (rather than saying something like 
```go
if val != expected {
   t.Fail()
}
```
you can say something like 
```go
So(val, ShouldEqual, expectedVal)
```
), with several keywords (`ShouldContain`, `ShouldBeNil`, `ShouldNotBeNil`, etc.), as well as the availability of the context for testing, so that anonymous functions (say in a test httpHandler at test time) can make assertions as well. 

That said, the built-in test framework can definitely be used. My thought with the addition was mostly that it makes tests easier to reason about and maybe more approachable? If the aim is to minimize external dependencies, which I understand, I'd rewrite these with the standard testing library.